src/pages/waiting-room/cat-b/cat-b-waiting-room.scss

```diff
@@ -1,4 +1,4 @@
-page-waiting-room {
+page-cat-b-waiting-room {
```

Comment:
So - i think the filename could be waiting-room-b.scss

with class page-waiting-room-b